style.css

```diff
@@ -99,3 +98,377 @@ section {
+  font-weight: 700;
+  font-size: 34px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text-title {
+  display: flex;
+  flex-direction: row;
+  align-self: flex-start;
+  padding: 0;
+  gap: 12px;
+  align-items: center;
+}
+
+.client {
+  height: 14px;
+  font-style: normal;
+  font-weight: 600;
+  text-decoration: none;
+  font-size: 15px;
+  line-height: 16px;
+  margin: 0;
+  color: #344563;
+}
+
+.client-skills {
+  font-weight: 600;
+  font-size: 14px;
+  margin: 0;
+  line-height: 17px;
+  font-style: normal;
+  text-decoration: none;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  color: #7a869a;
+}
+
+.dot {
+  height: 8px;
+  width: 8px;
+  border-radius: 50%;
+  background-color: #c1c7d0;
+  display: inline-block;
+}
+
+.period {
+  font-size: 14px;
+  font-weight: 600;
+  margin: 0;
+  line-height: 16px;
+  font-style: normal;
+  text-decoration: none;
+  color: #7a869a;
+}
+
+.content-text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  color: #344563;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  padding: 0;
+  width: 203px;
+  height: 25px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  list-style: none;
+}
+
+.style {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  text-align: center;
+  line-height: 15px;
+  letter-spacing: 0.03em;
+  padding: 5px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+.style:hover {
+  transform: scale(1.3);
+  transition-duration: 0.3s;
+  background-color: #6070ff;
+  color: white;
+}
+
+.link {
+  padding: 13px;
+  gap: 10px;
+  left: 0;
+  border: 1px solid #6070ff;
+  border-radius: 9px;
+  letter-spacing: 0.04em;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 26px;
+  text-decoration: none;
+  text-align: center;
+  cursor: pointer;
+  order: 3;
+  color: #396df2;
+  margin: 0;
+}
+
+.link:hover {
+  transform: scale(1.3);
+  transition-duration: 0.3s;
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.link:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.About-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 115px 25px;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10;
+  gap: 12px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin: 0;
+}
+
+.about-head {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin: 0;
+}
+
+.about-skills {
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 464.15px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
```

Comment:
Kindly observe the margins between the content the edge of the page in the about section.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219922601-48e3b1fc-6216-4271-acce-b8bf3c5951ae.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219922644-d543766c-bd61-4753-bb1c-ea71626b8625.png)

Reply:
I try everything to fixed it, but when I open it in full web page the both sides are equal.